### PR TITLE
Fix: Key conflicts in German localization

### PR DESCRIPTION
**Pull request by xezon**
* Relates to #2091

This change fixes 99 key conflicts in German localization.

* The Boss Arm the Mob can now be researched with key O and no longer conflicts with Buggy Ammo (B).
* The Boss Gattling Tank can now be produced with key A and no longer conflicts with Avenger (G).
* The Boss Land Mines can now be researched with key M and no longer conflicts with Laser Missiles (L).
* The China Supply Center can now be placed with key U and no longer conflicts with Tunnel Network (N).
* The China Black Napalm can now be researched with key N and no longer conflicts with Listening Outpost (S).
* The China Subliminal Messaging can now be researched with key B and no longer conflicts with Uranium Shells (U).
* The China Nuclear Tanks can now be researched with key P and no longer conflicts with Nationalism (N).
* The China Carpet Bomb can now be placed with key T and no longer conflicts with Dozer (B).
* The China EMP Pulse can now be placed with key E and no longer conflicts with Tank Paradrop (P).
* The China Napalm Bomb Drop can now be triggered with key N and no longer conflicts with Guard (W).
* The China Nuke Bomb Drop can now be triggered with key N and no longer conflicts with Guard (W).
* The China Super Hacker can now be produced with key A and no longer conflicts with Super Lotus (S).
* The China Attack Outpost can now be produced with key S and no longer conflicts with View Command Center (H).
* The China Attack Helix can now be produced with key X and no longer conflicts with View Command Center (H).
* The China Fortified Bunker can now be placed with key B and no longer conflicts with Barracks (K).
* The GLA Tunnel Network can now be placed with key N and no longer conflicts with Speaker Tower (T).
* The GLA Tunnel Network can now be placed with key N and no longer conflicts with Speaker Tower (T).
* The GLA Stinger Site can now be placed with key T and no longer conflicts with Stop (S).
* The GLA Truck Bio Bomb can now be researched with key B and no longer conflicts with Stop (S).
* The GLA Toxin Shells can now be researched with key T and no longer conflicts with Stop (S).
* The GLA Ambush can now be placed with key I and no longer conflicts with Camo Netting (N).
* The GLA Sneak Attack can now be placed with key S and no longer conflicts with Ambush (N).
* The GLA Toxin Terrorist can now be produced with key B and no longer conflicts with Toxin Rebel (S).
* The GLA Toxin Tunnel can now be placed with key N and no longer conflicts with Stop (S).
* The USA Combat Drop can now be triggered with key N and no longer conflicts with Attack Move (A).
* The USA Hold The Line plan can now be set with key N and no longer conflicts with Stop (S).
* The USA Search and Destroy plan can now be set with key G and no longer conflicts with Carpet Bomb (T).
* The USA Patriot System can now be placed with key M and no longer conflicts with Bunker (B).
* The USA Strategy Center can now be placed with key T and no longer conflicts with Stop (S).
* The USA Burton can now be produced with key O and no longer conflicts with Missile Defender (B).
* The USA Nuke Cannon can now be produced with key K and no longer conflicts with Black Napalm (N).
* The USA Advanced Training can now be researched with key A and no longer conflicts with Particle Cannon (P).
* The USA King Raptor can now be produced with key R and no longer conflicts with Stealth Fighter (T).
* The USA Aurora Alpha can now be produced with key A and no longer conflicts with Bunker Buster (B).
* The USA Bunker Buster can now be produced with key B and no longer conflicts with Rally Point (U).
* The USA Combat Chinook can now be produced with key K and no longer conflicts with Chinook (C).

## Issue log

```
AmericaDozerCommandSet has key conflict with
  Command_ConstructAmericaStrategyCenter : CONTROLBAR:ConstructAmericaStrategyCenter "&Strategiezentrum"
  STOP : "S"

GLAWorkerCommandSet has key conflict with
  Command_ConstructGLAStingerSite : CONTROLBAR:ConstructGLAStingerSite "&Stingerstellung"
  STOP : "S"

GLAVehicleBombTruckCommandSet has key conflict with
  Command_UpgradeGLABombTruckBioBomb : CONTROLBAR:UpgradeGLABombTruckBioBomb "&Säurebomben"
  Command_Stop : CONTROLBAR:Stop "&Stopp"

GLAVehicleBombTruckCommandSet has key conflict with
  Command_UpgradeGLABombTruckBioBomb : CONTROLBAR:UpgradeGLABombTruckBioBomb "&Säurebomben"
  STOP : "S"

ChinaVehicleHelixCommandSet has key conflict with
  Command_ChinaHelixDropNapalmBomb : CONTROLBAR:DropNapalmBomb "&Werfen Sie eine Napalmbombe ab."
  Command_Guard : CONTROLBAR:Guard "Be&wachen"

ChinaHelixGattlingCannonCommandSet has key conflict with
  Command_ChinaHelixDropNapalmBomb : CONTROLBAR:DropNapalmBomb "&Werfen Sie eine Napalmbombe ab."
  Command_Guard : CONTROLBAR:Guard "Be&wachen"

ChinaHelixPropagandaTowerCommandSet has key conflict with
  Command_ChinaHelixDropNapalmBomb : CONTROLBAR:DropNapalmBomb "&Werfen Sie eine Napalmbombe ab."
  Command_Guard : CONTROLBAR:Guard "Be&wachen"

ChinaHelixBattleBunkerCommandSet has key conflict with
  Command_ChinaHelixDropNapalmBomb : CONTROLBAR:DropNapalmBomb "&Werfen Sie eine Napalmbombe ab."
  Command_Guard : CONTROLBAR:Guard "Be&wachen"

AmericaAirfieldCommandSet has key conflict with
  Command_UpgradeAmericaBunkerBusters : CONTROLBAR:UpgradeAmericaBunkerBusters "B&unker Buster"
  Command_SetRallyPoint : CONTROLBAR:SetRallyPoint "Sammelp&unkt"

AmericaBarracksCommandSet has key conflict with
  Command_ConstructAmericaInfantryMissileDefender : CONTROLBAR:ConstructAmericaInfantryMissileDefender "Raketen-&Bot"
  Command_ConstructAmericaInfantryColonelBurton : CONTROLBAR:ConstructAmericaInfantryColonelBurton "Kommando&bot"

AmericaStrategyCenterCommandSet has key conflict with
  Command_InitiateBattlePlanHoldTheLine : CONTROLBAR:InitiateBattlePlanHoldTheLine "Schlachtplan \"&Stellung halten\"
  Command_StrategyCenter_Stop : CONTROLBAR:Stop "&Stopp"

ChinaCommandCenterCommandSet has key conflict with
  Command_ConstructChinaDozer : CONTROLBAR:ConstructChinaDozer "&Bulldozer"
  Early_Command_ChinaCarpetBomb : CONTROLBAR:CarpetBomb "&Bombenteppich"

ChinaCommandCenterCommandSetUpgrade has key conflict with
  Command_ConstructChinaDozer : CONTROLBAR:ConstructChinaDozer "&Bulldozer"
  Early_Command_ChinaCarpetBomb : CONTROLBAR:CarpetBomb "&Bombenteppich"

ChinaWarFactoryCommandSet has key conflict with
  Command_ConstructChinaVehicleListeningOutpost : CONTROLBAR:ConstructChinaVehicleListeningOutpost "Horchpo&sten"
  Command_UpgradeChinaBlackNapalm : CONTROLBAR:UpgradeChinaBlackNapalm "&Schwarzes Napalm"

ChinaWarFactoryCommandSetUpgrade has key conflict with
  Command_ConstructChinaVehicleListeningOutpost : CONTROLBAR:ConstructChinaVehicleListeningOutpost "Horchpo&sten"
  Command_UpgradeChinaBlackNapalm : CONTROLBAR:UpgradeChinaBlackNapalm "&Schwarzes Napalm"

GLACommandCenterCommandSet has key conflict with
  Command_Ambush : CONTROLBAR:Ambush "Hi&nterhalt"
  Command_SneakAttack : CONTROLBAR:SneakAttack "Schleicha&ngriff"

GLAPalaceCommandSet has key conflict with
  Command_UpgradeGLAToxinShells : CONTROLBAR:UpgradeGLAToxinShells "&Säuregranaten"
  Command_Stop : CONTROLBAR:Stop "&Stopp"

GC_Chem_GLAWorkerCommandSet has key conflict with
  GC_Chem_Command_ConstructGLAStingerSite : CONTROLBAR:ConstructGLAStingerSite "&Stingerstellung"
  GC_Chem_Command_ConstructGLATunnelNetwork : CONTROLBAR:Chem_ConstructGLATunnelNetwork "&Säure-Tunnelsystem"

GC_Chem_GLAWorkerCommandSet has key conflict with
  GC_Chem_Command_ConstructGLAStingerSite : CONTROLBAR:ConstructGLAStingerSite "&Stingerstellung"
  STOP : "S"

GC_Chem_GLAWorkerCommandSet has key conflict with
  GC_Chem_Command_ConstructGLATunnelNetwork : CONTROLBAR:Chem_ConstructGLATunnelNetwork "&Säure-Tunnelsystem"
  STOP : "S"

GC_Chem_GLABarracksCommandSet has key conflict with
  GC_Chem_Command_ConstructGLAInfantryRebel : CONTROLBAR:Chem_ConstructGLAInfantryRebel "&Säure-Cyborg"
  GC_Chem_Command_ConstructGLAInfantryTerrorist : CONTROLBAR:Chem_ConstructGLAInfantryTerrorist "&Säure-Bombe"

GC_Chem_GLACommandCenterCommandSet has key conflict with
  GC_Chem_Command_Ambush : CONTROLBAR:Ambush "Hi&nterhalt"
  Command_SneakAttack : CONTROLBAR:SneakAttack "Schleicha&ngriff"

GC_Slth_GLAWorkerCommandSet has key conflict with
  GC_Slth_Command_ConstructGLAStingerSite : CONTROLBAR:ConstructGLAStingerSite "&Stingerstellung"
  STOP : "S"

GC_Slth_GLACommandCenterCommandSet has key conflict with
  Command_Ambush : CONTROLBAR:Ambush "Hi&nterhalt"
  Command_SneakAttack : CONTROLBAR:SneakAttack "Schleicha&ngriff"

AirF_AmericaDozerCommandSet has key conflict with
  AirF_Command_ConstructAmericaStrategyCenter : CONTROLBAR:ConstructAmericaStrategyCenter "&Strategiezentrum"
  STOP : "S"

AirF_AmericaStrategyCenterCommandSet has key conflict with
  Command_InitiateBattlePlanBombardment : CONTROLBAR:InitiateBattlePlanBombardment "Schlachtplan \"&Bombardement\"
  AirF_Command_CarpetBomb : CONTROLBAR:CarpetBomb "&Bombenteppich"

AirF_AmericaStrategyCenterCommandSet has key conflict with
  Command_InitiateBattlePlanHoldTheLine : CONTROLBAR:InitiateBattlePlanHoldTheLine "Schlachtplan \"&Stellung halten\"
  Command_StrategyCenter_Stop : CONTROLBAR:Stop "&Stopp"

AirF_AmericaBarracksCommandSet has key conflict with
  AirF_Command_ConstructAmericaInfantryMissileDefender : CONTROLBAR:ConstructAmericaInfantryMissileDefender "Raketen-&Bot"
  AirF_Command_ConstructAmericaInfantryColonelBurton : CONTROLBAR:ConstructAmericaInfantryColonelBurton "Kommando&bot"

AirF_AmericaAirfieldCommandSet has key conflict with
  AirF_Command_ConstructAmericaJetRaptor : CONTROLBAR:ConstructAmericaJetKingRaptor "King Rap&tor"
  AirF_Command_ConstructAmericaJetStealthFighter : CONTROLBAR:ConstructAmericaJetStealthFighter "&Tarnkappenbomber"

AirF_AmericaAirfieldCommandSet has key conflict with
  Command_UpgradeAmericaBunkerBusters : CONTROLBAR:UpgradeAmericaBunkerBusters "B&unker Buster"
  Command_SetRallyPoint : CONTROLBAR:SetRallyPoint "Sammelp&unkt"

AirF_AmericaVehicleChinookCommandSet has key conflict with
  Command_CombatDrop : CONTROLBAR:CombatDrop "Einheiten &absetzen"
  Command_AttackMove : CONTROLBAR:AttackMove "&Angreifen"

AirF_AmericaSupplyCenterCommandSet has key conflict with
  AirF_Command_ConstructAFGChinook : CONTROLBAR:ConstructAmericaVehicleChinook "&Chinook"
  AirF_Command_ConstructAmericaVehicleChinook : CONTROLBAR:AirF_ConstructAmericaVehicleChinook "&Chinook"

Demo_GLAWorkerCommandSet has key conflict with
  Demo_Command_ConstructGLAStingerSite : CONTROLBAR:ConstructGLAStingerSite "&Stingerstellung"
  STOP : "S"

Demo_GLACommandCenterCommandSet has key conflict with
  Demo_Command_Ambush : CONTROLBAR:Ambush "Hi&nterhalt"
  Command_SneakAttack : CONTROLBAR:SneakAttack "Schleicha&ngriff"

Demo_GLAWorkerCommandSetUpgrade has key conflict with
  Demo_Command_ConstructGLAStingerSite : CONTROLBAR:ConstructGLAStingerSite "&Stingerstellung"
  STOP : "S"

Demo_GLACommandCenterCommandSetUpgrade has key conflict with
  Demo_Command_Ambush : CONTROLBAR:Ambush "Hi&nterhalt"
  Command_SneakAttack : CONTROLBAR:SneakAttack "Schleicha&ngriff"

Slth_GLAWorkerCommandSet has key conflict with
  Slth_Command_ConstructGLAStingerSite : CONTROLBAR:ConstructGLAStingerSite "&Stingerstellung"
  STOP : "S"

Slth_GLACommandCenterCommandSet has key conflict with
  Command_Ambush : CONTROLBAR:Ambush "Hi&nterhalt"
  Command_SneakAttack : CONTROLBAR:SneakAttack "Schleicha&ngriff"

Slth_GLACommandCenterCommandSet has key conflict with
  Command_Ambush : CONTROLBAR:Ambush "Hi&nterhalt"
  Command_UpgradeGLACamoNetting : CONTROLBAR:UpgradeGLACamoNetting "Tarn&netze"

Slth_GLACommandCenterCommandSet has key conflict with
  Command_SneakAttack : CONTROLBAR:SneakAttack "Schleicha&ngriff"
  Command_UpgradeGLACamoNetting : CONTROLBAR:UpgradeGLACamoNetting "Tarn&netze"

Slth_GLAVehicleBombTruckCommandSet has key conflict with
  Command_UpgradeGLABombTruckBioBomb : CONTROLBAR:UpgradeGLABombTruckBioBomb "&Säurebomben"
  Command_Stop : CONTROLBAR:Stop "&Stopp"

Slth_GLAVehicleBombTruckCommandSet has key conflict with
  Command_UpgradeGLABombTruckBioBomb : CONTROLBAR:UpgradeGLABombTruckBioBomb "&Säurebomben"
  STOP : "S"

Chem_GLAWorkerCommandSet has key conflict with
  Chem_Command_ConstructGLAStingerSite : CONTROLBAR:ConstructGLAStingerSite "&Stingerstellung"
  Chem_Command_ConstructGLATunnelNetwork : CONTROLBAR:Chem_ConstructGLATunnelNetwork "&Säure-Tunnelsystem"

Chem_GLAWorkerCommandSet has key conflict with
  Chem_Command_ConstructGLAStingerSite : CONTROLBAR:ConstructGLAStingerSite "&Stingerstellung"
  STOP : "S"

Chem_GLAWorkerCommandSet has key conflict with
  Chem_Command_ConstructGLATunnelNetwork : CONTROLBAR:Chem_ConstructGLATunnelNetwork "&Säure-Tunnelsystem"
  STOP : "S"

Chem_GLABarracksCommandSet has key conflict with
  Chem_Command_ConstructGLAInfantryRebel : CONTROLBAR:Chem_ConstructGLAInfantryRebel "&Säure-Cyborg"
  Chem_Command_ConstructGLAInfantryTerrorist : CONTROLBAR:Chem_ConstructGLAInfantryTerrorist "&Säure-Bombe"

Chem_GLAVehicleBombTruckCommandSet has key conflict with
  Command_UpgradeGLABombTruckBioBomb : CONTROLBAR:UpgradeGLABombTruckBioBomb "&Säurebomben"
  Command_Stop : CONTROLBAR:Stop "&Stopp"

Chem_GLAVehicleBombTruckCommandSet has key conflict with
  Command_UpgradeGLABombTruckBioBomb : CONTROLBAR:UpgradeGLABombTruckBioBomb "&Säurebomben"
  STOP : "S"

Chem_GLACommandCenterCommandSet has key conflict with
  Chem_Command_Ambush : CONTROLBAR:Ambush "Hi&nterhalt"
  Command_SneakAttack : CONTROLBAR:SneakAttack "Schleicha&ngriff"

Nuke_ChinaWarFactoryCommandSet has key conflict with
  Nuke_Command_ConstructChinaVehicleListeningOutpost : CONTROLBAR:ConstructChinaVehicleListeningOutpost "Horchpo&sten"
  Command_UpgradeChinaBlackNapalm : CONTROLBAR:UpgradeChinaBlackNapalm "&Schwarzes Napalm"

Nuke_ChinaWarFactoryCommandSetUpgrade has key conflict with
  Nuke_Command_ConstructChinaVehicleListeningOutpost : CONTROLBAR:ConstructChinaVehicleListeningOutpost "Horchpo&sten"
  Command_UpgradeChinaBlackNapalm : CONTROLBAR:UpgradeChinaBlackNapalm "&Schwarzes Napalm"

Nuke_ChinaVehicleHelixCommandSet has key conflict with
  Nuke_Command_ChinaHelixDropNukeBomb : CONTROLBAR:DropNukeBomb "&Werfen Sie eine Atombombe ab."
  Command_Guard : CONTROLBAR:Guard "Be&wachen"

Nuke_ChinaHelixGattlingCannonCommandSet has key conflict with
  Nuke_Command_ChinaHelixDropNukeBomb : CONTROLBAR:DropNukeBomb "&Werfen Sie eine Atombombe ab."
  Command_Guard : CONTROLBAR:Guard "Be&wachen"

Nuke_ChinaHelixPropagandaTowerCommandSet has key conflict with
  Nuke_Command_ChinaHelixDropNukeBomb : CONTROLBAR:DropNukeBomb "&Werfen Sie eine Atombombe ab."
  Command_Guard : CONTROLBAR:Guard "Be&wachen"

Nuke_ChinaHelixBattleBunkerCommandSet has key conflict with
  Nuke_Command_ChinaHelixDropNukeBomb : CONTROLBAR:DropNukeBomb "&Werfen Sie eine Atombombe ab."
  Command_Guard : CONTROLBAR:Guard "Be&wachen"

SupW_AmericaDozerCommandSet has key conflict with
  SupW_Command_ConstructAmericaStrategyCenter : CONTROLBAR:ConstructAmericaStrategyCenter "&Strategiezentrum"
  STOP : "S"

SupW_AmericaBarracksCommandSet has key conflict with
  SupW_Command_ConstructAmericaInfantryMissileDefender : CONTROLBAR:ConstructAmericaInfantryMissileDefender "Raketen-&Bot"
  SupW_Command_ConstructAmericaInfantryColonelBurton : CONTROLBAR:ConstructAmericaInfantryColonelBurton "Kommando&bot"

SupW_AmericaStrategyCenterCommandSet has key conflict with
  Command_InitiateBattlePlanHoldTheLine : CONTROLBAR:InitiateBattlePlanHoldTheLine "Schlachtplan \"&Stellung halten\"
  Command_StrategyCenter_Stop : CONTROLBAR:Stop "&Stopp"

SupW_AmericaAirfieldCommandSet has key conflict with
  Command_UpgradeAmericaBunkerBusters : CONTROLBAR:UpgradeAmericaBunkerBusters "B&unker Buster"
  Command_SetRallyPoint : CONTROLBAR:SetRallyPoint "Sammelp&unkt"

Infa_ChinaDozerCommandSet has key conflict with
  Infa_Command_ConstructChinaBarracks : CONTROLBAR:ConstructChinaBarracks "Cyborg-&Kaserne"
  Infa_Command_ConstructChinaBunker : CONTROLBAR:Infa_ConstructChinaBunker "Befestigter Bun&ker"

Infa_ChinaCommandCenterCommandSet has key conflict with
  Infa_Command_ConstructChinaDozer : CONTROLBAR:ConstructChinaDozer "&Bulldozer"
  Early_Command_ChinaCarpetBomb : CONTROLBAR:CarpetBomb "&Bombenteppich"

Infa_ChinaCommandCenterCommandSetUpgrade has key conflict with
  Infa_Command_ConstructChinaDozer : CONTROLBAR:ConstructChinaDozer "&Bulldozer"
  Early_Command_ChinaCarpetBomb : CONTROLBAR:CarpetBomb "&Bombenteppich"

Infa_ChinaBarracksCommandSet has key conflict with
  Infa_Command_ConstructChinaInfantryHacker : CONTROLBAR:Infa_ConstructChinaInfantryHacker "&Super-Hacker"
  Infa_Command_ConstructChinaInfantryBlackLotus : CONTROLBAR:Infa_ConstructChinaInfantryBlackLotus "&Super-Bot"

Infa_ChinaBarracksCommandSetUpgrade has key conflict with
  Infa_Command_ConstructChinaInfantryHacker : CONTROLBAR:Infa_ConstructChinaInfantryHacker "&Super-Hacker"
  Infa_Command_ConstructChinaInfantryBlackLotus : CONTROLBAR:Infa_ConstructChinaInfantryBlackLotus "&Super-Bot"

Infa_ChinaWarFactoryCommandSet has key conflict with
  Infa_Command_ConstructChinaVehicleListeningOutpost : CONTROLBAR:Infa_ConstructChinaVehicleListeningOutpost "&Horchposten"
  VIEW_COMMAND_CENTER : "H"

Infa_ChinaWarFactoryCommandSetUpgrade has key conflict with
  Infa_Command_ConstructChinaVehicleListeningOutpost : CONTROLBAR:Infa_ConstructChinaVehicleListeningOutpost "&Horchposten"
  VIEW_COMMAND_CENTER : "H"

Infa_ChinaAirfieldCommandSet has key conflict with
  Infa_Command_ConstructChinaVehicleHelix : CONTROLBAR:Infa_ConstructChinaVehicleHelix "&Helix"
  VIEW_COMMAND_CENTER : "H"

Infa_ChinaAirfieldCommandSetUpgrade has key conflict with
  Infa_Command_ConstructChinaVehicleHelix : CONTROLBAR:Infa_ConstructChinaVehicleHelix "&Helix"
  VIEW_COMMAND_CENTER : "H"

Infa_ChinaHelixBombCommandSet has key conflict with
  Command_ChinaHelixDropNapalmBomb : CONTROLBAR:DropNapalmBomb "&Werfen Sie eine Napalmbombe ab."
  Command_Guard : CONTROLBAR:Guard "Be&wachen"

Lazr_AmericaDozerCommandSet has key conflict with
  Lazr_Command_ConstructAmericaStrategyCenter : CONTROLBAR:ConstructAmericaStrategyCenter "&Strategiezentrum"
  STOP : "S"

Lazr_AmericaStrategyCenterCommandSet has key conflict with
  Command_InitiateBattlePlanHoldTheLine : CONTROLBAR:InitiateBattlePlanHoldTheLine "Schlachtplan \"&Stellung halten\"
  Command_StrategyCenter_Stop : CONTROLBAR:Stop "&Stopp"

Lazr_AmericaBarracksCommandSet has key conflict with
  Lazr_Command_ConstructAmericaInfantryMissileDefender : CONTROLBAR:ConstructAmericaInfantryMissileDefender "Raketen-&Bot"
  Lazr_Command_ConstructAmericaInfantryColonelBurton : CONTROLBAR:ConstructAmericaInfantryColonelBurton "Kommando&bot"

Lazr_AmericaAirfieldCommandSet has key conflict with
  Command_UpgradeAmericaBunkerBusters : CONTROLBAR:UpgradeAmericaBunkerBusters "B&unker Buster"
  Command_SetRallyPoint : CONTROLBAR:SetRallyPoint "Sammelp&unkt"

Tank_ChinaCommandCenterCommandSet has key conflict with
  Tank_Command_ConstructChinaDozer : CONTROLBAR:ConstructChinaDozer "&Bulldozer"
  Command_ChinaCarpetBomb : CONTROLBAR:CarpetBomb "&Bombenteppich"

Tank_ChinaCommandCenterCommandSet has key conflict with
  Tank_Command_TankParadrop : CONTROLBAR:TankParadrop "&Panzer-Abwurf"
  Command_EMPPulse : CONTROLBAR:EMPPulse "EM&P-Angriff"

Tank_ChinaCommandCenterCommandSetUpgrade has key conflict with
  Tank_Command_ConstructChinaDozer : CONTROLBAR:ConstructChinaDozer "&Bulldozer"
  Command_ChinaCarpetBomb : CONTROLBAR:CarpetBomb "&Bombenteppich"

Tank_ChinaCommandCenterCommandSetUpgrade has key conflict with
  Tank_Command_TankParadrop : CONTROLBAR:TankParadrop "&Panzer-Abwurf"
  Command_EMPPulse : CONTROLBAR:EMPPulse "EM&P-Angriff"

Tank_ChinaWarFactoryCommandSet has key conflict with
  Tank_Command_ConstructChinaVehicleListeningOutpost : CONTROLBAR:ConstructChinaVehicleListeningOutpost "Horchpo&sten"
  Command_UpgradeChinaBlackNapalm : CONTROLBAR:UpgradeChinaBlackNapalm "&Schwarzes Napalm"

Tank_ChinaWarFactoryCommandSetUpgrade has key conflict with
  Tank_Command_ConstructChinaVehicleListeningOutpost : CONTROLBAR:ConstructChinaVehicleListeningOutpost "Horchpo&sten"
  Command_UpgradeChinaBlackNapalm : CONTROLBAR:UpgradeChinaBlackNapalm "&Schwarzes Napalm"

Boss_AmericaDozerCommandSet has key conflict with
  Boss_Command_ConstructChinaBunker : CONTROLBAR:ConstructChinaBunker "&Bunker"
  Boss_Command_ConstructAmericaPatriotBattery : CONTROLBAR:ConstructAmericaPatriotBattery "Patriot-&Batterie"

Boss_AmericaDozerCommandSet has key conflict with
  Boss_Command_ConstructGLATunnelNetwork : CONTROLBAR:ConstructGLATunnelNetwork "&Tunnelsystem"
  Boss_Command_ConstructChinaSpeakerTower : CONTROLBAR:ConstructChinaSpeakerTower "Propaganda&turm"

Boss_ChinaCommandCenterCommandSet has key conflict with
  Boss_Command_ConstructChinaDozer : CONTROLBAR:ConstructBossDozer "&Bulldozer"
  Command_ChinaCarpetBomb : CONTROLBAR:CarpetBomb "&Bombenteppich"

Boss_ChinaCommandCenterCommandSetUpgrade has key conflict with
  Boss_Command_ConstructChinaDozer : CONTROLBAR:ConstructBossDozer "&Bulldozer"
  Command_ChinaCarpetBomb : CONTROLBAR:CarpetBomb "&Bombenteppich"

Boss_ChinaAirfieldCommandSet has key conflict with
  Command_UpgradeAmericaLaserMissiles : CONTROLBAR:UpgradeAmericaLaserMissiles "Laser-&Lenkraketen"
  Command_UpgradeChinaMines : CONTROLBAR:UpgradeChinaMines "&Landminen"

Boss_ChinaBarracksCommandSet has key conflict with
  Boss_Command_ConstructChinaInfantryBlackLotus : CONTROLBAR:ConstructChinaInfantryBlackLotus "&Black Bot"
  Boss_Command_ConstructAmericaInfantryColonelBurton : CONTROLBAR:ConstructAmericaInfantryColonelBurton "Kommando&bot"

Boss_ChinaBarracksCommandSet has key conflict with
  Command_UpgradeAmericaRangerFlashBangGrenade : CONTROLBAR:UpgradeAmericaFlashBangGrenade "B&lendgranaten"
  Command_UpgradeChinaMines : CONTROLBAR:UpgradeChinaMines "&Landminen"

Boss_ChinaBarracksCommandSetUpgrade has key conflict with
  Boss_Command_ConstructChinaInfantryBlackLotus : CONTROLBAR:ConstructChinaInfantryBlackLotus "&Black Bot"
  Boss_Command_ConstructAmericaInfantryColonelBurton : CONTROLBAR:ConstructAmericaInfantryColonelBurton "Kommando&bot"

Boss_ChinaWarFactoryCommandSet has key conflict with
  Boss_Command_ConstructAmericaVehicleAvenger : CONTROLBAR:ConstructAmericaTankAvenger "Aven&ger"
  Boss_Command_ConstructChinaTankGattling : CONTROLBAR:ConstructChinaTankGattling "&Gatling-Panzer"

Boss_ChinaWarFactoryCommandSet has key conflict with
  Boss_Command_ConstructAmericaVehicleSentryDrone : CONTROLBAR:ConstructAmericaVehicleSentryDrone "Überwachung&sdrohne"
  Command_UpgradeChinaBlackNapalm : CONTROLBAR:UpgradeChinaBlackNapalm "&Schwarzes Napalm"

Boss_ChinaWarFactoryCommandSetUpgrade has key conflict with
  Boss_Command_ConstructAmericaVehicleAvenger : CONTROLBAR:ConstructAmericaTankAvenger "Aven&ger"
  Boss_Command_ConstructChinaTankGattling : CONTROLBAR:ConstructChinaTankGattling "&Gatling-Panzer"

Boss_ChinaWarFactoryCommandSetUpgrade has key conflict with
  Boss_Command_ConstructAmericaVehicleSentryDrone : CONTROLBAR:ConstructAmericaVehicleSentryDrone "Überwachung&sdrohne"
  Command_UpgradeChinaBlackNapalm : CONTROLBAR:UpgradeChinaBlackNapalm "&Schwarzes Napalm"

Boss_AmericaParticleUplinkCannonCommandSet has key conflict with
  Command_FireParticleUplinkCannon : CONTROLBAR:FireParticleUplinkCannon "&Partikelkanone abfeuern"
  Command_UpgradeAmericaAdvancedTraining : CONTROLBAR:UpgradeAmericaAdvancedTraining "S&pezialtraining"

Boss_AmericaParticleUplinkCannonCommandSetUpgrade has key conflict with
  Command_FireParticleUplinkCannon : CONTROLBAR:FireParticleUplinkCannon "&Partikelkanone abfeuern"
  Command_UpgradeAmericaAdvancedTraining : CONTROLBAR:UpgradeAmericaAdvancedTraining "S&pezialtraining"

Boss_GLAScudStormCommandSet has key conflict with
  Command_UpgradeGLABuggyAmmo : CONTROLBAR:UpgradeGLABuggyAmmo "&Buggy-Munition"
  Command_UpgradeGLAArmTheMob : CONTROLBAR:UpgradeGLAArmTheMob "Cyborg-Trupp &bewaffnen"

Boss_GLAScudStormCommandSetUpgrade has key conflict with
  Command_UpgradeGLABuggyAmmo : CONTROLBAR:UpgradeGLABuggyAmmo "&Buggy-Munition"
  Command_UpgradeGLAArmTheMob : CONTROLBAR:UpgradeGLAArmTheMob "Cyborg-Trupp &bewaffnen"

Boss_ChinaNuclearMissileCommandSet has key conflict with
  Boss_Command_UpgradeChinaNationalism : CONTROLBAR:UpgradeChinaNationalism "&Nationalismus"
  Command_UpgradeChinaNuclearTanks : CONTROLBAR:UpgradeChinaNuclearTanks "&Nuklearpanzer"

Boss_ChinaNuclearMissileCommandSet has key conflict with
  Command_UpgradeChinaSubliminalMessaging : CONTROLBAR:UpgradeChinaSubliminalMessaging "&Unterbewusste Beeinflussung"
  Command_UpgradeChinaUraniumShells : CONTROLBAR:UpgradeChinaUraniumShells "&Urangranaten"

Boss_ChinaNuclearMissileCommandSetUpgrade has key conflict with
  Boss_Command_UpgradeChinaNationalism : CONTROLBAR:UpgradeChinaNationalism "&Nationalismus"
  Command_UpgradeChinaNuclearTanks : CONTROLBAR:UpgradeChinaNuclearTanks "&Nuklearpanzer"

Boss_ChinaNuclearMissileCommandSetUpgrade has key conflict with
  Command_UpgradeChinaSubliminalMessaging : CONTROLBAR:UpgradeChinaSubliminalMessaging "&Unterbewusste Beeinflussung"
  Command_UpgradeChinaUraniumShells : CONTROLBAR:UpgradeChinaUraniumShells "&Urangranaten"
```

### CODE REVIEW

**Comment by commy2** [comment]
IIrc (it has been decades), W is not a conflict in DE, because the "select air" button doesn't work at all in DE.

Also, iirc, the Y and Z hotkeys are flipped. If Y is the hotkey, you have to press Z and vice versa.

File: Patch104pZH/GameFilesEdited/Data/generals.str [hunk 1]
```diff
@@ -20182,9 +20221,11 @@ RU: "Следопыт"
 AR: "تارﺎﺴﻤﻟا ﺚﺣﺎﺑ"
 END
 
+// Patch104p @tweak 13/07/2023 Changes DE from "Patriot-Batterie".
+
 OBJECT:PatriotBattery
 US: "Patriot Missile System"
-DE: "Patriot-Batterie"
+DE: "Patriot-Batterie System"
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/generals.str:20228]: Wouldn't it be Batteriesystem?

**Comment by xezon** [comment]
W is Guard Mode (Be&wachen)

**Comment by commy2** [comment]
Ah, would be neat if DE had a W equivalent.

**Comment by xezon** [comment]
Some other languages have no Select Aircraft mapping either.

File: Patch104pZH/GameFilesEdited/Data/generals.str (hunk 1 above)
Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/generals.str:20228]: Not sure. For reference, this is EMP Battery name:

```
US: "E&MP Patriot System"
DE: "EMP Patriot-System"
```

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/generals.str:20228]: Perhaps just call it Patriot-System ?

**Comment by commy2** [comment]
They all should get it. Same for arrow keys map movement. Cycling through units could be added to EN, but some off side buttons.

File: Patch104pZH/GameFilesEdited/Data/generals.str (hunk 1 above)
Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/generals.str:20228]: Done

**Comment by xezon** [comment]
Documentation added. Looks good.

**Pull request merged**